config.c

```diff
@@ -60,36 +67,38 @@ load_config(const char *fileName)
             if (c == NULL)
                 die(NULL, "Expected value for argument %s\n", arg);
 
-            if (!strcmp(arg, "min_scale")) {
+            if (!strcmp(arg, "min_scale"))
                 conf.min_scale = strtof(c, NULL);
-            } else if (!strcmp(arg, "max_scale")) {
+            else if (!strcmp(arg, "max_scale"))
                 conf.max_scale = strtof(c, NULL);
-            } else if (!strcmp(arg, "scroll_speed")) {
+            else if (!strcmp(arg, "scroll_speed"))
                 conf.scroll_speed = strtof(c, NULL);
-            } else if (!strcmp(arg, "drag_friction")) {
+            else if (!strcmp(arg, "drag_friction"))
                 conf.drag_friction = strtof(c, NULL);
-            } else if (!strcmp(arg, "scale_friction")) {
+            else if (!strcmp(arg, "scale_friction"))
                 conf.scale_friction = strtof(c, NULL);
-            } else if (!strcmp(arg, "key_move_speed")) {
+            else if (!strcmp(arg, "key_move_speed"))
                 conf.key_move_speed = strtof(c, NULL);
-            } else if (!strcmp(arg, "windowed")) {
+            else if (!strcmp(arg, "windowed"))
                 switch (c[0]) {
                 case 'f':
                 case 'F':
+                case 'n':
+                case 'N':
                 case '0':
                     conf.windowed = false;
                     break;
                 case 't':
                 case 'T':
+                case 'y':
+                case 'Y':
                 case '1':
                     conf.windowed = true;
                     break;
                 default:
                     break;
```

Comment:
This logic is separated into parse_bool in master, so when i merge ill move it there 🤝🤝

Reply:
💀